### DISCUSSION TITLE
which version to do .kcr files?

### DISCUSSION THREAD

**Discussion by George2777** [noDRM/DeDRM_tools#108] [General]
I have Calibre on a Mac with the original DeDRM_tools_7.2.1 and it's not recognizing the .kcr files. What version of everything do I need (and is there a pointer for how to install it into Calibre) to be able to convert my .kcr files? thanks in advance.

**Comment by ElleKayEm** [comment]
If you don't have an E-ink Kindle, you'll need no later then Kindle for Mac 1.31 to download your books.  See this post for getting books that are decryptable: https://www.mobileread.com/forums/showthread.php?t=283371

Might as well use the latest (as of now) calibre 6.1 and DeDRM 10.0.3 (unless you have an old Mac).

**Reply by George2777** [reply]
is this a permanent situation or are forthcoming versions of DeDRM likely to enable this with a modern Kindle app version?

**Reply by ElleKayEm** [reply]
The last two updates when DeDRM was updated to work with the then current version of Kindle for PC/Mac, Amazon broke it within 24 hours.

**Reply by JSWolf** [reply]
> is this a permanent situation or are forthcoming versions of DeDRM likely to enable this with a modern Kindle app version?

You could buy a cheap Kindle and register that to be able to download direct from Amazon using your web browser.
